docs/operate-and-deploy/installation/upgrading.md

```diff
@@ -137,6 +137,24 @@ This will stop all processing and delete any internal topics in Kafka.
 
 ## Upgrade notes
 
+### Upgrading from ksqlDB 0.14.0 to 0.20.0
```

Comment:
why 0.14.0 ?

Reply:
Just followed the pattern from the below upgrading paths. I don't know if just show 0.19 -> 0.20 and use the latest path show below.